components/dashboard/sections/Vendors.tsx

```diff
@@ -404,6 +405,7 @@ const Vendors = ({ accountSlug }: DashboardSectionProps) => {
         <StyledModal onClose={closeDrawer} width="570px">
           <VendorForm
             host={host}
+            supportsTaxForm={host.requiredLegalDocuments.includes('US_TAX_FORM')}
```

Comment:
Should we extract this from host inside `VendorForm`?

Reply:
I hesitated between the two, but went with this option as there's no guarantee that `host.requiredLegalDocuments` will be fetched before being passed to `VendorForm`. Sounded safer to do it right from where it's retrieved.